llvm/lib/IR/IntrinsicInst.cpp

```diff
@@ -599,6 +599,19 @@ Intrinsic::ID VPIntrinsic::getForOpcode(unsigned IROPC) {
   return Intrinsic::not_intrinsic;
 }
 
+Intrinsic::ID VPIntrinsic::getForIntrinsic(Intrinsic::ID Id) {
```

Comment:
Could this be made `constexpr`? I see you're using it with a constant `RecurKind::FAdd` later on.

Reply:
741d6c0eba6b8fa6e2b0e75dd6ec332836dfc8f6